packages/bundler-plugin-core/src/index.ts

```diff
@@ -363,9 +367,30 @@ function generateGlobalInjectorCode({
       _global.SENTRY_RELEASES["${key}"]={id:"${release}"};`;
   }
 
+  if (injectBuildInformation) {
+    const buildInfo = getBuildInformation();
+
+    code += `
+      _global.SENTRY_BUILD_INFO=${JSON.stringify(buildInfo)}`;
```

Comment:
h: Can we add a semicolon at the end here? Although rare, there are some situations where a missing semicolon will mess up user code.

Reply:
oops, definitely!